whois-update/src/test/java/net/ripe/db/whois/update/dns/zonemaster/StartDomainTestRequestTest.java

```diff
@@ -0,0 +1,42 @@
+package net.ripe.db.whois.update.dns.zonemaster;
+
+import net.ripe.db.whois.common.rpsl.AttributeType;
+import net.ripe.db.whois.common.rpsl.RpslAttribute;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import net.ripe.db.whois.update.dns.DnsCheckRequest;
+import net.ripe.db.whois.update.dns.zonemaster.domain.StartDomainTestRequest;
+import net.ripe.db.whois.update.domain.Update;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class StartDomainTestRequestTest {
+
+    private StartDomainTestRequest startDomainTestRequest;
+    @Mock
+    Update update;
+
+
+    @Test
+    public void remove_last_dot_from_domain(){
+        isDomainType();
+        assertEquals("18.182.5.in-addr.arpa", startDomainTestRequest.getParams().getDomain());
+    }
+
```

Comment:
I personally prefer integration tests for this. Is it possible with zonemaster? I thiunk there must be existing one we can look for

Reply:
with the zone master IT you are testing the output of zone master, there you will be not able to see if the dot has been deleted or not.

Reply:
I created a IT but I think that is not providing any value.

Reply:
I have added a comment on how to properly test it with existing integration test class. I do not mind just junit test for this but if we are using the existing class for testing other functionality for zonemaster lets keep it the same  :) Honestly in a way we are mocking the request anyways

Reply:
Created the IT and fix the dummy server error